### ISSUE TITLE
Reconsider using dot in front of asset ids. 

### ISSUE THREAD

**Issue by Jo-Schie** [mapme-initiative/mapme.biodiversity#20]
Currently, the asset ids are named `.assetid`. This however might create a problem when you use `dplyr` pipes  because a dot `.` indicates that the pipe should return the object from the pipe.  I'm not sure if this is really an issue an did not test it as well, nevertheless i wondered whether it was really necessary the dot in front. What is the rationale @goergen95 ?

**Comment by goergen95** [comment]
Indeed, its not strictly necessary with the current approach. Its more a relic of the idea to create a `portfolio` class and keep the `.assetid` hidden from the user through the processing chain. Since we are still in very early development, it would not cause major friction to change it now. However, I did not experience any issues with a column name preceded by a dot in pipes. In my understanding you would refer to the previous output by a dot alone, e.g. `.["assetid"]`. Do you have an example where it leads to issues?

**Comment by Jo-Schie** [comment]
No, i do not have an example... it was more a guess, that it might cause problems at some point in time. but I do not really see an advantage of the dot in front and think it is rather confusing the user. Not sure...if it creates too much work because you'd have to change the documentatoin and everything....if thats the case then let's maybe label it as `ìmprovement` and put it in the backlog.

**Comment by goergen95** [comment]
I just get 4 matches for `.assetid` in the source code, so I guess it will not be too much trouble. I'll quickly check locally if all test pass. Just note that if we change it now and you continue to process your LA portfolio you will have to take care that more recent results will not have the dot...
